projects/packages/sync/changelog/fix-sync-limit-post-content

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: added
+
+Jetpack Sync :: Impose a max limit of 5MB on post_content that can be synced.
```

Comment:
Since this changelog is in the Sync package itself, the prefix is probably redundant.
```suggestion
Impose a max limit of 5MB on post_content that can be synced.
```

Since you're also changing Jetpack in this PR, you'll need a similar file in `projects/plugins/jetpack/changelog/`. In that case, [writing-a-good-changelog-entry.md](https://github.com/Automattic/jetpack/blob/master/docs/writing-a-good-changelog-entry.md) suggests the prefix should be `Jetpack Sync:` rather than `Jetpack Sync ::`.